app/main.py

```
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    tokens = command.split()
+    if len(tokens) != 3:
+        return
+    if tokens[0] != "cp":
+        return
+    source_file_name = tokens[1]
+    destination_file_name = tokens[2]
+    if source_file_name == destination_file_name:
+        return
+
+    try:
+        with (open(source_file_name, "r") as file_out,
+              open(destination_file_name, "w") as file_in):
+            file_in.write(file_out.read())
+    except FileNotFoundError:
+        print("File not found")
```

This leaves a debug print in committed code and violates the Clean Code guidance: "Add comments, prints, and functions to check your solution when you write your code. Don't forget to delete them when you are ready to commit and push your code." Remove the `print("File not found")`. If you want to suppress the traceback, either `return` silently or let the exception propagate; do not leave debug prints.